sqewer.gemspec

```diff
@@ -49,6 +48,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "sqlite3"
   spec.add_development_dependency "dotenv"
   spec.add_development_dependency "simplecov"
+  spec.add_development_dependency "nokogiri"
```

Comment:
how come we need this now?

Reply:
https://github.com/WeTransfer/sqewer/actions/runs/3411182918/jobs/5675066902

```
RuntimeError:
  Unable to find a compatible xml library. Ensure that you have installed or added to your Gemfile one of ox, oga, libxml, nokogiri or rexml
# ./spec/spec_helper.rb:8:in `require'
# ./spec/spec_helper.rb:8:in `<top (required)>'
# ./spec/sqewer_module_spec.rb:1:in `require_relative'
# ./spec/sqewer_module_spec.rb:1:in `<top (required)>'
```

aws-sdk-sqs needs it for ruby 3+ as runtime dependency. We don't have to make it sqewer dependency and lock it, because the client has the freedom to choose their own xml lib

Reply:
👍🏼